### PR TITLE
Cd

### PR DESCRIPTION
**Pull request by hcvdwerf**

### CODE REVIEW

File: .github/workflows/release-package.yml [hunk 1]
```diff
@@ -17,20 +19,38 @@ jobs:
...
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build-and-push-image
+    steps:
+      - name: Lowercase the repo name
+        run: echo "REPO_LOWER=$(echo $GITHUB_REPOSITORY | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
```
Comment (brunopacheco1) [inline .github/workflows/release-package.yml:48]: If you don't need this variable for anything else, can you please remove and add explicitly the image tag on line 56?

File: .github/workflows/release-package.yml [hunk 2]
```diff
@@ -17,20 +19,38 @@ jobs:
...
         with:
           registry: ${{ env.REGISTRY }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+          
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
```
Comment (brunopacheco1) [inline .github/workflows/release-package.yml:34]: Perhaps you can reuse this variable or a new variable combined link:
`IMAGE_NAME: ${{ env.REGISTRY }}/${{ github.repository }}`

And reuse it every where.

File: .github/workflows/release-package.yml (hunk 1 above)
Comment (hcvdwerf) [reply .github/workflows/release-package.yml]: check

File: .github/workflows/release-package.yml (hunk 2 above)
Comment (hcvdwerf) [reply .github/workflows/release-package.yml]: check

**Review by brunopacheco1 — DISMISSED** [review]

**Pull request merged**